Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -133,7 +145,19 @@ internal void SetNetworkInstanceId(uint newNetId)
         }
 
         // only used when fixing duplicate scene IDs during post-processing
-        public void ForceSceneId(uint newSceneId) => m_SceneId = newSceneId;
+        public void ForceSceneId(uint newSceneId)
+        {
+            m_SceneId = newSceneId;
+
+#if UNITY_EDITOR
+            if (!Application.isPlaying && (ThisIsAPrefab() || ThisIsASceneObjectWithPrefabParent(out GameObject prefab)))
+            {
+                Debug.Log("NetworkIdentity: Recording modifications");
+                PrefabUtility.RecordPrefabInstancePropertyModifications(this);
```

Comment:
why is this using the prefab utility exactly?
sceneIds should only be for scene objects

Reply:
The scene object is a prefab that has NetworkIdentities inside of it. (Loot zones, Windows and Doors that are networked).

Reply:
If this method is not called, in 2018.3, the new sceneId value is not saved to disk and remains zero.  This breaks this implementation if it cannot persist the sceneId.

Reply:
ok